examples/javascript/button.js

```diff
@@ -26,7 +26,7 @@
 var groveSensor = require('jsupm_grove');
```

Comment:
Hi, I think this is the real bug - instead of loading jsupm_grove, this example should be loading jsupm_button.

Can you give that a try and see if that works with the original Button(0) below?